docs/jsonapi-spec.md

```diff
@@ -592,6 +593,87 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
 
 If an error occurs the command will not return `data`.
 
+
+### findCollection Command
```

Comment:
this whole thing belongs to that section above `Collections` and not `Documents`